server/app.py

```diff
@@ -94,14 +94,15 @@ def update_cache():
         new_stops.append(stop_info)
 
     for group in grouped_stops:
-        stop_info = get_stop_predictions(group.get('ids'), group.get('operator'), group.get('group_name'), group.get('use_destination_as_name', False))
+        for stop in group.get("stops"):
```

Comment:
we can do this, that way if stops is null, we wont get the below error
```suggestion
        for stop in group.get("stops", []):
```

```
Traceback (most recent call last):
  File "/server/app.py", line 216, in helper_thread
    update_cache()
  File "/server/app.py", line 97, in update_cache
    for stop in group.get("stops"):
TypeError: 'NoneType' object is not iterable
```